lib/ssl/src/ssl_handshake.erl

```diff
@@ -1973,7 +1979,7 @@ int_to_bin(I) ->
 
 %% The end-entity certificate provided by the client MUST contain a
 %% key that is compatible with certificate_types.
-certificate_types(Version) when Version =< {3,3} ->
+certificate_types(Version) when Version =< ?'TLS-1.2' ->
```

Comment:
Can not be TLS-1.3

Reply:
so this change is correct because it states that the version must be less or equal than TLS-1.2?